storage.go

```diff
@@ -620,18 +620,20 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 	o.ID = rp
 	o.Path = path
 
-	metadata := output.Metadata
-	if target, ok := metadata["x-amz-meta-bs-link-target"]; ok {
-		// The path is a symlink object.
-		if !s.features.VirtualLink {
-			err = NewOperationNotImplementedError("virtual_link")
-			return nil, err
-		}
+	if output.Metadata != nil {
+		metadata := output.Metadata
+		if target, ok := metadata["X-Amz-Meta-Bs-Link-Target"]; ok {
```

Comment:
Maybe we can use it as a const?

Reply:
The situation here is the same as that of qingstor.

Reply:
Hi, please read s3 sdk's doc.

```go
	// By default unmarshaled keys are written as a map keys in following canonicalized format:
	// the first letter and any letter following a hyphen will be capitalized, and the rest as lowercase.
	// Set `aws.Config.LowerCaseHeaderMaps` to `true` to write unmarshaled keys to the map as lowercase.
```

I think both input and output should in lowercase.

Reply:
OK, got it!